test_notebooks/time_variable_Deff.py

```diff
@@ -0,0 +1,89 @@
+"""
+A script for calculating Deff based on MSD data from t=0 to a user-selected
+timepoint or range of timepoints (in which case Deff is a geometric mean).
+"""
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+import scipy
+from scipy import stats
+
+from bokeh.plotting import figure, show, gridplot
+import zipfile
+
+
+ALPHA = 1
+
+
+zf = zipfile.ZipFile('brain-diffusion_data.zip')
+file_handle = zf.open('Mean_Square_Displacement_Data.csv')
+msd = pd.read_csv(file_handle)
+# Rename columns without biological replicate number (this allows for
+# averaging of biological replicates later)
+msd.columns = msd.iloc[0]
+msd = msd.iloc[1:]
+# Create a list of column names without replicates
+columns = msd.columns
+columns = columns[1:67]
+columns2 = [ii for n, ii in enumerate(columns) if ii not in columns[:n]]
+
+
+"""
+The following function combines MSDs of three biological replicates
```

Comment:
Importantly, this documentation should also explain what kind of format the inputs and outputs are.  That isn't specified clearly.  Is df a Numpy array?  Pandas? Does it need to be a certain length or width?  How does the function know which columns to use?  It also doesn't specifically tell me what the variable "ColumnName" is.  That would all be very helpful here.